spec/api/v1/schemas.rb

```diff
@@ -31,6 +31,7 @@ module Schemas
         links: LINKS,
         benchmark: BENCHMARK,
         profile: PROFILE,
+        profile_create: PROFILE_CREATE,
```

Comment:
The schema accepted for profile creation should be identical to the schema returned by a GET request. Some attributes are not able to be set, but the entire set of attributes should be allowed (and are allowed).